.github/workflows/test-and-check.yml

```diff
@@ -14,7 +14,8 @@ jobs:
     steps:
       - run: curl -X POST https://devprod-status-bot.devprod.workers.dev/pr-project/workers-sdk/${{ github.event.number }}
 
-  check:
+  check-all:
```

Comment:
Be aware that we can't make this a required check since it would never pass on any PRs.
So this would only be for information only.

It should not run on `ubuntu-latest` since the `Checks` is a required check to land the Version Packages PR.